Tools/ardupilotwaf/git_submodule.py

```diff
@@ -111,8 +112,28 @@ def u(s):
     def __str__(self):
         return 'Submodule update: %s' % self.submodule
 
+def check_folder(path_list=[]):
```

Comment:
```suggestion
def module_is_uninitialised(path):
```

Do a single module at a time.